feature-libs/product-configurator/rulebased/testing/configurator-test-data.ts

```diff
@@ -416,7 +416,7 @@ export const productConfigurationWithoutSelectedOptions: Configurator.Configurat
       issueNavigationDone: true,
     },
   };
-export const ov_group_description = 'Group 1';
+export const OV_GROUP_DESCRIPTION = 'Group 1';
```

Comment:
Is this compliant to regex ^[_$A-Za-z][$A-Za-z0-9]*$|^[_$A-Z][_$A-Z0-9]+$.  ?
From the description I understand that "_" is not allowed inside a variable name